server/adaptors/integrations/__data__/repository/apache/apache-1.0.0.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "apache",
+    "version": "1.0.0",
+    "displayName": "Apache Dashboard",
+    "description": "Apache web logs collector",
+    "license": "Apache-2.0",
+    "type": "logs",
+    "author": "OpenSearch",
+    "sourceUrl": "https://github.com/elastic/beats/tree/main/filebeat/module/apache/_meta/kibana/7/search",
```

Comment:
source is our own repo for the integration:
`    "sourceUrl": "https://github.com/opensearch-project/dashboards-observability/tree/main/server/adaptors/integrations/__data__/repository/aws_s3/info",
`